shell/ci/release/dryrun.sh

```diff
@@ -34,16 +34,21 @@ git checkout "$CIRCLE_BRANCH"
 
 # Squash our branch onto the HEAD (default) branch to mimic
 # what would happen after merge.
-git merge --squash "$OLD_CIRCLE_BRANCH"
-git commit -m "$COMMIT_MESSAGE"
-
-GH_TOKEN="$(cat "$HOME/.outreach/github.token")"
-if [[ -z $GH_TOKEN ]]; then
-  echo "Failed to read Github personal access token" >&2
+if ! git diff --quiet "$OLD_CIRCLE_BRANCH" ; then
+    git merge --squash "$OLD_CIRCLE_BRANCH"
+    git commit -m "$COMMIT_MESSAGE"
+    GH_TOKEN="$(cat "$HOME/.outreach/github.token")"
```

Comment:
Nit: we should use `gh` instead of the legacy path.
```suggestion
    GH_TOKEN="$(gh auth token)"
```